include/fluent-bit/flb_avro.h

```diff
@@ -31,7 +31,7 @@
 #define MEMORY_POOL_MINIMUM_SIZE sizeof(void *)
 
 struct flb_avro_fields {
-    flb_sds_t schema_id;
+    int schema_id;
```

Comment:
Disclaimer: Not familiar with kafka/avro requirements

It seems like the current format is indeed wrong because it sends 16 bytes instead of 4 specified in the docs. If the expected format for schema_id is a 4 byte integer in hex format, shouldn't we leave this as a string?

Reply:
In confluent document(https://docs.confluent.io/platform/current/schema-registry/index.html#schema-id-allocation)

> Schema ID allocation always happens in the primary node and Schema IDs are always monotonically increasing.

and in schema registry REST API

> Response JSON Object:
> - subject (string) – Name of the subject that this schema is registered under
> - id (int) – Globally unique identifier of the schema
> - version (int) – Version of the returned schema
> - schema (string) – The schema string

schema registry use id as int.